programs/lz4cli.c

```diff
@@ -49,7 +49,7 @@
 #  pragma warning(disable : 4127)      /* disable: C4127: conditional expression is constant */
 #endif
 
-#define _POSIX_SOURCE 1        /* for fileno() within <stdio.h> on unix */
+#define _XOPEN_VERSION 600 /* POSIX.2001, for fileno() within <stdio.h> on unix */
```

Comment:
Could you please provide a source which justify the betterness of this detection macro ?